src/ducks/resource.js

```diff
@@ -33,75 +33,58 @@ const resourceReducer = (state = initialState, action) => {
 };
 
 // Action Creators
-const fetchResource = (id, type) => {
+function fetchResource(id, type) {
   type = type || 'projects';
-  return (dispatch) => {
+  return (dispatch, getState) => {
+    const project = getState().project.data;
```

Comment:
Sorry but is there a code example for this? I've not used either of those methods before.